now.json

```
@@ -1,7 +1,7 @@
 {
   "rewrites": [
     {
-      "source": "/",
+      "source": "/(api/v1/frameworks)?",
```

Is the capture group necessary? I think this would work:

```suggestion
      "source": "/api/v1/frameworks",
```

---

@TooTallNate but we want the group to be optional